canvas_modules/common-canvas/src/common-properties/controls/someofselect/someofselect.jsx

```diff
@@ -45,10 +45,10 @@ class SomeofselectControl extends React.Component {
 	}
 
 	// this is needed in order to reset the property value when a value is filtered out.
-	updateValueFromFilterEnum(skipValidateInput) {
+	updateValueFromFilterEnum(skipValidateInput, onMount) {
```

Comment:
instead of onMount should this be `updateType` or something like that.  onMount seems like a boolean value.